episode-6.msrt

```diff
@@ -254,7 +252,7 @@
 51
 00:03:43,310 --> 00:03:47,310
 [ru] Зелье Зомбификации... глупости какие!
-[en] A potion of Zombification... what nonsense!
+[en] A potion of Zombification... that's nothing!
```

Comment:
"What a nonsense" is definitely grammatically incorrect. Nonsense is an uncountable noun so it should not have an article in front of it ([When to use articles before nouns](http://www.quickanddirtytips.com/education/grammar/when-to-use-articles-before-nouns))